doc-content/jbpm-docs/src/main/asciidoc/BPMN2/create-user-task-proc.adoc

```diff
@@ -19,7 +19,7 @@ User tasks are used to include human actions as input to the business process.
 |Description
 
 | *Name*
-| The display name of the business rule task.
+| The name of the user task. You can also double-click the shape to edit the name of the user task. 
```

Comment:
Same question as above, which shape we have to double-click?

Reply:
Done